setup.md

```diff
@@ -4,166 +4,241 @@ This document will guide you through setting up Fennel on your
-graphics and sounds yourself inside TIC, while LOVE lets you import
-from external sources and use any resolution. However, the cost of
-this flexibility is that it's a lot more complicated. Both tools offer
-cross-platform support across Windows, Mac, and Linux systems, but
-TIC-80 games can be played in the browser and LOVE games cannot.
+LOVE is a game-making framework for the Lua programming
+language. Because Fennel compiles to Lua, you can reference the [LOVE
+wiki](https://love2d.org/wiki/Main_Page) when making games with Fennel.
+LOVE is more flexible than TIC-80 in that it allows you to import from
+external resources and use any resolution or memory you like, but at
+a cost in that it is more complicated to make games in.
 
-## Using Fennel in TIC-80
+Both TIC-80 and LOVE offer cross-platform support across Windows, Mac,
+and Linux systems, but TIC-80 games can be played in the browser and
+LOVE games cannot.
+
+This section consists of the following subsections:
 
-Support for Fennel is built-in. If you want to use the built-in text
-editor, you don't need any other tools, just launch TIC-80 and run
-`new fennel` to get started.
+* [Using Fennel in TIC-80](#using-fennel-in-tic-80)
+* [Using Fennel with LOVE](#using-fennel-with-love)
 
-But if you want to see an example, this
-[Conway's Life](https://tic.computer/play?cart=656) implementation
-could be a good starting point. Click start, press ESC, use the arrows
-to go down to "close game", and press Z to go to the console. Then
-press ESC to see the source.
+## Using Fennel in TIC-80
+
+Support for Fennel is built into TIC-80. If you want to use the
+built-in text editor, you don't need any other tools, just launch
+TIC-80 and run `new fennel` to get started.
 
-The [TIC-80 wiki](https://github.com/nesbox/TIC-80/wiki) is an
-indispensable documentation resource.
+For references , see the Links below:
 
-If you would prefer to use an external editor, this
-[project skeleton repo](https://github.com/stefandevai/fennel-tic80-game)
-provides some helpful support.
+* [Conway's Life](https://tic.computer/play?cart=656): Implementing
+  this would be a good learning exercise.
+    * Click "start" to begin
+    * Press the Esc key to open a menu
+    * Use the arrow keys to navigate the menu
+    * Press the Z key to open the console, followed by Esc to see the
+      source code.
+* [The TIC-80 wiki](https://github.com/nesbox/TIC-80/wiki)
+* [project skeleton repo](https://github.com/stefandevai/fennel-tic80-game) for
+  information on using external editors, instead of the built-in
+  TIC-80 editor.
 
 ## Using Fennel with LOVE
 
-LOVE has no built-in support for Fennel, so you need to bring it yourself.
-This [project skeleton repo](https://gitlab.com/alexjgriffith/min-love2d-fennel)
-for LOVE shows you how to do that, including a console-based REPL for
-debugging and reloading.
+LOVE has no built-in support for Fennel, so setup support yourself.
```

Comment:
"so you'll need to set up support yourself" maybe?

Reply:
Hahaha, thanks for catching this.